### PR TITLE
Add cocureency ingestion per threads setting

### PR DESCRIPTION
**Pull request by atlas-comstock**

### CODE REVIEW

File: internal/storage/disk/disk.go [hunk 1]
```diff
@@ -96,15 +96,15 @@ func (s *Storage) Open(dir string, options config.Badger) error {
 	}
 
 	// max size of lsm tree in bytes after which data is propagated to disk.
-	// The default is 64 MB.
+	// The default is 2 MB.
```
Comment (kelindar) [inline internal/storage/disk/disk.go:99]: How does it perform?

**Review by kelindar — APPROVED** [review]

File: internal/storage/disk/disk.go (hunk 1 above)
Comment (atlas-comstock) [reply internal/storage/disk/disk.go:99]: Oh, in fact, this maybe a typo before. If you check the default of badger, it is 2MB. @kelindar

Comment (atlas-comstock) [reply internal/storage/disk/disk.go:99]: I just update the comment.

Comment (atlas-comstock) [reply internal/storage/disk/disk.go:99]: I won't introduce breaking changes or compatibility problems if possible

**Comment by atlas-comstock** [comment]
Fix #100

**Pull request merged**